apps/hash-frontend/package.json

```diff
@@ -122,7 +122,7 @@
     "@types/jsonpath": "0.2.0",
     "@types/lodash": "4.14.188",
     "@types/papaparse": "5.3.14",
-    "@types/react": "18.2.79",
+    "@types/react": "18.2.68",
```

Comment:
Downgraded due to some issue later ones have introduced, can check back on it again later